test/framework/src/main/java/org/opensearch/test/OpenSearchIntegTestCase.java

```diff
@@ -2602,6 +2656,29 @@ public static Settings buildRemoteStoreNodeAttributes(
         );
     }
 
+    public static Settings buildRemoteStoreNodeAttributes(
+        String segmentRepoName,
+        Path segmentRepoPath,
+        String translogRepoName,
+        Path translogRepoPath,
+        String clusterStateRepoName,
+        Path clusterStateRepoPath,
```

Comment:
Same here, lets name variables according to their intended usage to avoid confusion.

Reply:
Addressed.